types/es6-shim/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "dependencies": {
+    "es6-shim": "0.34"
```

Comment:
I followed the check list: "Test the change in your own code. (Compile and run.)"
To run the tests, I needed to install `es6-shim`, that is the reason for the `package.json`. Shall it be removed?